jmix-pivottable/pivottable-flowui/src/main/java/io/jmix/pivottableflowui/export/PivotTableExporterImpl.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.pivottableflowui.export;
+
+import com.google.common.collect.Sets;
+import io.jmix.core.common.util.Preconditions;
+import io.jmix.flowui.download.Downloader;
+import io.jmix.pivottableflowui.component.PivotTable;
+import io.jmix.pivottableflowui.export.model.PivotData;
+import io.jmix.pivottableflowui.kit.component.model.Renderer;
+import org.springframework.lang.Nullable;
+
+import java.util.Collections;
+import java.util.Set;
+
+public class PivotTableExporterImpl implements PivotTableExporter {
```

Comment:
Considering interface and naming, It looks like it should be a bean, but...

**Update 1:** in classic it has a different name, which makes it reasonable. Convert to bean